aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -799,21 +810,9 @@ impl AptosVM {
 
         let func =
             code_storage.load_script(serialized_script.code(), serialized_script.ty_args())?;
-        let script = match func.owner() {
-            LoadedFunctionOwner::Script(script) => script,
-            LoadedFunctionOwner::Module(_) => {
-                // This should not be reachable because loading a function from the script should
-                // set the owner correctly.
-                let msg = "Function loaded from script cannot come from module".to_string();
-                let err = PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR)
-                    .with_message(msg)
-                    .finish(Location::Undefined)
-                    .into_vm_status();
-                return Err(err);
-            },
-        };
 
         // Check that unstable bytecode cannot be executed on mainnet and verify events.
+        let script = func.expect_script()?;
```

Comment:
expect has the other connotation, maybe we can name this owner_as_script or smt?

Reply:
Renamed as `owner_as_script`